esmvaltool/recipes/ref/recipe_model_benchmarking_annual_cycle_region.yml

```diff
@@ -0,0 +1,149 @@
+  authors:
+    - lauer_axel
+    - bock_lisa
+    - hassler_birgit
+    - lindenlaub_lukas
+    - schlund_manuel
+    - weigel_katja
+  maintainer:
+    - lauer_axel
+  references:
+    - lauer25gmd
+  projects:
+    - dlrmabak
+
+
+datasets:
+  # Dataset to be benchmarked
+  - {dataset: MIROC6, grid: gn, benchmark_dataset: true, plot_label: 'MIROC6'}
+
+preprocessors:
+  pp_abs:
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    climate_statistics:
+      period: monthly
+    area_statistics:
+      operator: mean
+
+  pp_abs_200:
+    extract_levels:
+      levels: 20000
+      scheme: linear
+      coordinate: air_pressure
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    climate_statistics:
+      period: monthly
+    area_statistics:
+      operator: mean
+
+
+diagnostics:
+  anncyc:
+    description: Plot global mean mean annual cycle for the given period including a reference dataset.
+    variables:
+      tas: &var_default_abs
+        start_year: 1980
+        end_year: 2009
+        preprocessor: pp_abs
+        project: CMIP6
+        mip: Amon
+        exp: historical
+        ensemble: r1i1p1f1
+        grid: gr
+        reference_dataset: HadCRUT5
+        additional_datasets:
+          - {dataset: HadCRUT5, project: OBS, type: ground, version: 5.0.1.0-analysis,
+             tier: 2, reference_for_metric: true'}
+      pr:
+        <<: *var_default_abs
+        reference_dataset: GPCP-SG
+        additional_datasets:
+          - {dataset: GPCP-SG, project: OBS, type: atmos, version: 2.3, tier: 2,
+             reference_for_metric: true'}
+      psl:
+        <<: *var_default_abs
+        reference_dataset: ERA5
+        additional_datasets:
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3, reference_for_metric: true'}
+      hus_200:
+        <<: *var_default_abs
+        short_name: hus
+        reference_dataset: ERA5
+        preprocessor: pp_abs_200
+        additional_datasets:
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3, reference_for_metric: true'}
+      ua_200:
+        <<: *var_default_abs
+        short_name: ua
+        reference_dataset: ERA5
+        preprocessor: pp_abs_200
+        additional_datasets:
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3, reference_for_metric: true}
+    scripts:
+      allplots:
+        script: monitor/multi_datasets.py
+        plot_folder: '{plot_dir}'
+        group_variables_by: variable_group
+        plots:
+          annual_cycle:
+            annual_mean_kwargs: False
+            plot_kwargs:
+              MIROC6:
```

Comment:
I think it would be better if you use the "default" here for the models. Then all the models will have the same red line, and we do not have to hardcode the model name in the recipe for all the times when the recipe is run for the REF.

Reply:
Thanks, changed